api/swagger/forms.yaml

```diff
@@ -67,6 +66,36 @@ paths:
           $ref: '#/components/responses/RequestTimeout'
         '500':
           $ref: '#/components/responses/InternalServer'
+  /data/{id}:
+    get:
+      operationId: getFormData
+      tags:
+        - Get Form Data
+      description: Gets the form data
+      parameters:
+        - "$ref": "#/components/parameters/formId"
+        - in: query
+          name: dataRef
+          description: Unique identifier of the data to be prefilled. This is returned by the submit API.
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Form Data
+          content:
+            application/json:
+              schema:
+                "$ref" : '#/components/schemas/PrefillResponse'
+        '400':
+          $ref: '#/components/responses/GenericBadRequest'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '408':
+          $ref: '#/components/responses/RequestTimeout'
+        '500':
+          $ref: '#/components/responses/InternalServer'
+        default:
+          "$ref": "#/components/responses/default"
   /submit/{id}:
```

Comment:
we should remove prefer header for now from the public API as we dont support it

Reply:
similarly 202 response should be removed as it is not yet supported in submit